operator/watchers/node.go

```diff
@@ -76,19 +71,28 @@ func (nodeGetter) ListK8sSlimNode() []*slim_corev1.Node {
 }
 
 // nodesInit starts up a node watcher to handle node events.
-func nodesInit(wg *sync.WaitGroup, slimClient slimclientset.Interface, stopCh <-chan struct{}) {
+func nodesInit(wg *sync.WaitGroup, slimClient slimclientset.Interface, stopCh <-chan struct{}, logger *slog.Logger) {
 	nodeSyncOnce.Do(func() {
+		nodeQueue = workqueue.NewTypedRateLimitingQueueWithConfig[string](
+			workqueue.NewTypedMaxOfRateLimiter(
+				workqueue.NewTypedItemExponentialFailureRateLimiter[string](1*time.Second, 30*time.Second),
+				&workqueue.TypedBucketRateLimiter[string]{Limiter: rate.NewLimiter(rate.Limit(10), 10)},
```

Comment:
okay, I've just dropped rate-limiter from it. 
Before we had retries:
1s,2s,3s,4s... retries on errors + unbounded concurrent controllers.
now we have exponential retries 1s, ... 120s + max concurrent of 10 workers. 

Worst case with 5k nodes and 120s backoff, we have ~40 events per second to process. 
(Of course with starting backoff at 1s it could be a bit worse initially, but then we have 10 concurrent workers + client-go qps limits).